apps/src/code-studio/components/pairing/StudentSelector.jsx

```diff
@@ -3,9 +3,31 @@ import React from 'react';
 import {studentsShape} from './types';
 import i18n from '@cdo/locale';
 
+const cyan = '#0094ca';
```

Comment:
Looks like this exists in `@cdo/apps/util/color` (apps/src/util/color.js). You should be able to go ahead and import it from there.